.gitignore

```diff
@@ -130,4 +130,8 @@ dist
 .pnp.*
 
 # next build file
-build/
\ No newline at end of file
+build/
+
+*.env*
+*credentials*
+yarn.lock
```

Comment:
yarn.lock should be versioned to prevent version "unlocking" and sudden build breaks.
```suggestion
```

Reply:
Fixed